app/views/content/life-as-a-teacher/chemistry/_article.html.erb

```diff
@@ -0,0 +1,130 @@
+<div class="row inset">
+  <section class="col col-720 col-space-l-top">
+    <h2 class="heading--box-blue">Why teach chemistry</h2>
+    <div class="inset">
+      <p>Chemistry underpins everything, so you'll be amazed at the questions your students will start asking. Leading hands-on lab experiments and creating engaging lessons will mean no two days are the same in the classroom.</p>
+      <p>In addition to the personal rewards of teaching chemistry, there are practical benefits too, including:</p>
+      <ul>
+        <li>a <a href="/life-as-a-teacher/teacher-pay-and-benefits competitive salary starting at <%= value :salaries_starting_minshortened %> "> (or higher in London)</li>
```

Comment:
@Sarah-DfE this hasn't worked! You are missing the </a>
this isn't right

```suggestion
        <li>a <a href="/life-as-a-teacher/teacher-pay-and-benefits> competitive salary starting at <%= value :salaries_starting_minshortened %> "> (or higher in London)</a></li>
```

Reply:
![image](https://github.com/user-attachments/assets/ff68bb78-52b7-43dc-9088-fb2246a90a7c)